persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -63,7 +63,7 @@ WHERE qualified_id = ?;
 
 updateTeamMemberUser:
 UPDATE User
-SET name = ?, handle = ?, email = ?, phone = ?, accent_id = ?, team = ?, preview_asset_id = ?, complete_asset_id = ?, bot_service = ?, incomplete_metadata = 0
+SET name = ?, handle = ?, email = ?, phone = ?, accent_id = ?, team = ?, connection_status = ?, preview_asset_id = ?, complete_asset_id = ?, bot_service = ?, incomplete_metadata = 0
```

Comment:
```suggestion
SET name = ?,
    handle = ?,
    email = ?,
    phone = ?,
    accent_id = ?,
    team = ?,
    connection_status = ?,
    preview_asset_id = ?,
    complete_asset_id = ?,
    bot_service = ?,
    incomplete_metadata = 0
```

Make it more readable by breaking the lines. Easier to find issues like this in the future :)